realtime/socketIOEmitter.js

```diff
@@ -27,7 +27,14 @@ module.exports = (io, key, mssgObj) => {
   }
   for (const nsp in io.nsps) {
     if (nsp && rtUtils.shouldIEmitThisObj(nsp, obj)) {
-      io.of(nsp).emit(key, JSON.stringify(mssgObj));
+      // obj contains only the newest values
+      io.of(nsp).emit(key, JSON.stringify({
+        key: {
+          new: {
+            obj,
+          },
+        },
+      }));
     }
```

Comment:
Yes, the changes are still coming in. Attached a picture for details.
<img width="950" alt="screen shot 2016-11-13 at 4 07 59 pm" src="https://cloud.githubusercontent.com/assets/17040750/20250004/6d510968-a9bb-11e6-81df-f101d1f7b272.png">